kratos/tests/test_model_part_io.py

```diff
@@ -4,13 +4,18 @@
 import KratosMultiphysics
 
 import os
+import sys
 
 
 def GetFilePath(fileName):
     return os.path.dirname(os.path.realpath(__file__)) + "/" + fileName
 
 
 class TestModelPartIO(KratosUnittest.TestCase):
+    
+    def setUp(self):        
+        if (sys.version_info < (3, 2)):
+            self.assertRaisesRegex = self.assertRaisesRegexp

```

Comment:
if we could hide it so that it works the same on python2.7 and python3+ it would be better, since we would avoid user-side ifs.

is that very complicated?